### DISCUSSION TITLE
moved my node to a Pi 4 8GB and cannot sync plus getting strange log errors

### DISCUSSION THREAD

**Discussion by nunoluis** [Chia-Network/chia-blockchain#5223] [Support]
I was farming without any problems on a mid level desktop and have moved my node to a raspberry Pi 4. I installed a fresh 1.1.5 version and added my keys, plus moved my .chia/mainnet/db and .chia/mainnet/wallet folders from the desktop to the same location in the Pi. 

Also changed the target_peer_count to 30.

I have been lagging in sync for a long time and just looked at the debug.log file and got a lot of this:
```
2021-05-16T20:56:35.909 full_node asyncio                 : ERROR    SSL error in data received
protocol: <asyncio.sslproto.SSLProtocol object at 0x7fa953aac8>
transport: <_SelectorSocketTransport fd=53 read=polling write=<idle, bufsize=0>>
Traceback (most recent call last):
  File "/usr/lib/python3.7/asyncio/sslproto.py", line 526, in data_received
    ssldata, appdata = self._sslpipe.feed_ssldata(data)
  File "/usr/lib/python3.7/asyncio/sslproto.py", line 207, in feed_ssldata
    self._sslobj.unwrap()
  File "/usr/lib/python3.7/ssl.py", line 767, in unwrap
    return self._sslobj.shutdown()
ssl.SSLError: [SSL: KRB5_S_INIT] application data after close notify (_ssl.c:2609)
2021-05-16T20:57:04.489 full_node asyncio                 : ERROR    SSL error in data received
protocol: <asyncio.sslproto.SSLProtocol object at 0x7f88e6eef0>
transport: <_SelectorSocketTransport fd=57 read=polling write=<idle, bufsize=0>>
Traceback (most recent call last):
  File "/usr/lib/python3.7/asyncio/sslproto.py", line 526, in data_received
    ssldata, appdata = self._sslpipe.feed_ssldata(data)
  File "/usr/lib/python3.7/asyncio/sslproto.py", line 207, in feed_ssldata
    self._sslobj.unwrap()
  File "/usr/lib/python3.7/ssl.py", line 767, in unwrap
    return self._sslobj.shutdown()
ssl.SSLError: [SSL: KRB5_S_INIT] application data after close notify (_ssl.c:2609)
```

I cannot sync and have no idea what is going on... Can someone help? Let me know what other info I could facilitate you with.
Thanks

**Comment by nunoluis** [comment]
Also, looking at my CPU usage with htop, looks like a lot is going on, but the only thing running on the Pi is the Chia GUI.
<img width="1192" alt="Screenshot 2021-05-16 at 20 54 59" src="https://user-images.githubusercontent.com/56304966/118409432-aa124c00-b68a-11eb-90d1-aff98ca04969.png">

**Comment by room101-dev** [comment]
Might be best to copy over everything from another .chia/mainnet folder on a machine

_[ When I say 'copy over' be careful to view your config file and make sure that its using the old stuff, generally it don't matter what you copy, if the config file is 'virgin' its going to rebuild a system from scratch. So you need to run "du -h" (linux) and make sure that its using the exact same path's for the 'copy run' to the new machine. Given that your 'sync' didn't take a short time, I would speculate your rasberry is building the blockchain/db from scratch. ]_

I recently from scratch built a new blockchain on a mining system ( not a plotter ), and it took 3-4 days, and I'm talking 100% NVME drives on this miner.

It used to take just a few hours from git-download (blockchain) to fully sync'd and running.

I fear that an rasberry, with a normal ssd or worst a HDD could take weeks if not longer to have it 'sync'd' & running

No matter what they say, ... like CHIA is for 'old computers' [ that didn't support +4gb files], or chia is for rasberrys, or 'easy to make plots', or 'low energy', its all MYTHOLOGY

I find that plotting is every bit as ASIC 'complex' as BTC mining, its the fastest CPUS with the most cores, and the largest ram disks ( 1TB on board DDR4 )  that make the fastest plots that win this game, how long before china has a CHIA box that spit's out 108GB plots? I think very soon.

My mining machine above is an i3, with 4gb memory, but like I said its got two onboard NVME top performance drives the actual blockchain uses little to no memory, using 'sudo iostat' I can see that building the sql/db is massive #1 resource grinder, not unlike 'plotting'

**Reply by nunoluis** [reply]
I agree with you that the plotting is very computing intensive. On the another hand, once netspace growth slows down overall, farming should lead to the expected outcome: less resources used to maintain a blockchain. It's just a matter of when the netspace will grow so much that buying hardrives is not a good investment; this should happen faster if more people that already have extra unused HHD space start to farm and use it - then no one will buy drives for this anymore.

But, back to the topic, my Pi4 was not rebuilding from scratch. Only the DB and WALLET folders were moved, not config, etc, I added my keys and changed the config file to adjust for the Pi4 (just reduce max peers). This way the blockchain would just need to catch a few dozens blocks from when I copied the folders. Unfortunately, after connecting to many peers it would remain on the same block for 5-10min at a time, then catch up a little almost like a burst of blocks, but still be maybe 20-30 away from the last block signed, then halt again while the blockchain kept moving on. At some point I managed to be fully synced, but only for a couple of blocks, until it "frooze" again - not updating any more blocks - and become massively out of sync. This went on and on.

I also noticed that most of the peers I connected to were somehow behind. Maybe 1 or 2 max were fully synced.

I then, logged off the Pi4 node and went back to my desktop - at this point, the desktop DB was much more behind in the chain. It took the desktop <1min to be fully synced, to have a large amount of fully synced peers connected to and had no problem staying in sync and receiving challenges.

Seeing how much CPU the Pi4 was using (often hitting 100% on all 4 cores), it seems that I either have something configured wrongly (hard to figure out what... I had done this early April and it was ok) or the Pi4 cannot keep up with the whole transaction sync.

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
